kolibri/core/assets/src/views/CoreBanner.vue

```diff
@@ -57,9 +58,22 @@
         bannerClosed: false,
       };
     },
+    created() {
+      document.addEventListener('focusin', this.focusChange);
+    },
+    beforeDestroy() {
+      document.removeEventListener('focusin', this.focusChange);
+    },
     methods: {
       toggleBanner() {
         this.bannerClosed = !this.bannerClosed;
+        this.previouslyFocusedElement.focus();
```

Comment:
Put a guard here, since it's possible that `this.previouslyFocusedElement` is undefined.